tests/integration/targets/openssh_keypair/tasks/main.yml

```diff
@@ -17,7 +17,30 @@
     backends: "{{ backends + ['cryptography'] }}"
   when: cryptography_version.stdout is version('3.0', '>=') and bcrypt_version.stdout is version('3.1.5', '>=')
 
-- include_tasks: ./impl.yml
+- include_tasks: ../tests/core.yml
```

Comment:
Out of curiosity: why not have all files in `tasks/`?

Reply:
No good reason outside of it matching what I did with [openssh_cert](https://github.com/ansible-collections/community.crypto/tree/main/tests/integration/targets/openssh_cert/tests).